package.json

```diff
@@ -68,7 +68,8 @@
     "react-hot-toast": "2.4.0",
     "react-syntax-highlighter": "15.5.0",
     "sharp": "0.31.1",
-    "swr": "1.3.0"
+    "swr": "1.3.0",
+    "xmlbuilder": "15.1.1"
```

Comment:
I have moved this logic into npm now - https://github.com/boxyhq/jackson/pull/700, xmlbuilder can be cleaned up from here